docker/Dockerfile

```diff
@@ -0,0 +1,79 @@
+#
+# Dockerfile for Miaou development
+#
+
+
+
+# Based on docker official ubuntu LTS image
+FROM ubuntu:14.04
+
+
+
+# Define Dockerfile maintainer
+MAINTAINER Oliver THEBAULT <contact@oliver-thebault.com>
+
+
+
+# Install Ubuntu miscellaneous packages
+RUN \
+	sed -i 's/# \(.*multiverse$\)/\1/g' /etc/apt/sources.list && \
+	apt-get update && \
+	apt-get -y upgrade && \
+	apt-get install -y build-essential && \
+	apt-get install -y software-properties-common && \
+	apt-get install -y git tcl8.5 nano vim man curl wget unzip htop xz-utils && \
+	rm -rf /var/lib/apt/lists/*
+
+
+
+# Install Python (used to build NPM dependencies)
+RUN \
+	apt-get update && \
+	apt-get install -y python python-dev python-pip python-virtualenv && \
+	rm -rf /var/lib/apt/lists/*
+
+
+
+# Install Ruby and SASS
+RUN \
+	apt-get update && \
+	apt-get install -y ruby ruby-dev ruby-bundler && \
+	rm -rf /var/lib/apt/lists/* && \
+	gem install sass
+
+
+
+# Install IOJS/NPM and globally install "nodemon", "uglify" and "buster"
+ENV IOJS_VERSION 2.2.1
+RUN \
+	curl https://iojs.org/dist/v$IOJS_VERSION/iojs-v$IOJS_VERSION-linux-x64.tar.xz | tar -xJC /usr/src
```

Comment:
I'm not a fan of this. Can you instead add the repo to /etc/apt/sources.list.d/, import the gpg key, and run apt-get install?